bitnami/harbor/README.md

```diff
@@ -0,0 +1,275 @@
+
+## Introduction
+
+This [Helm](https://github.com/kubernetes/helm) chart installs [Harbor](https://github.com/goharbor/harbor) in a Kubernetes cluster. Welcome to [contribute](CONTRIBUTING.md) to Helm Chart for Harbor.
+
+## Prerequisites
+
+- Kubernetes cluster 1.10+
+- Helm 2.8.0+
+
+## Installation
+
+### Download the chart
+
+Download Harbor helm chart
+
+```bash
+git clone https://github.com/bitnami/charts
+```
+
+Change directory to Harbor code
+
+```bash
+cd charts/bitnami/harbor
+```
+
+### Configure the chart
+
+The following items can be configured in `values.yaml` or set via `--set` flag during installation.
+
+#### Configure the way how to expose Harbor service:
+
+- **Ingress**: The ingress controller must be installed in the Kubernetes cluster.
+  **Notes:** if the TLS is disabled, the port must be included in the command when pulling/pushing images. Refer to issue [#5291](https://github.com/goharbor/harbor/issues/5291) for the detail.
+- **ClusterIP**: Exposes the service on a cluster-internal IP. Choosing this value makes the service only reachable from within the cluster.
+- **NodePort**: Exposes the service on each Node’s IP at a static port (the NodePort). You’ll be able to contact the NodePort service, from outside the cluster, by requesting `NodeIP:NodePort`.
+- **LoadBalancer**: Exposes the service externally using a cloud provider’s load balancer.
+
+#### Configure the external URL
+
+The external URL for Harbor core service is used to:
+
+1. populate the docker/helm commands showed on portal
+2. populate the token service URL returned to docker/notary client
+
+Format: `protocol://domain[:port]`. Usually:
+
+- if expose the service via `Ingress`, the `domain` should be the value of `service.ingress.hosts.core`
+- if expose the service via `ClusterIP`, the `domain` should be the value of `service.clusterIP.name`
+- if expose the service via `NodePort`, the `domain` should be the IP address of one Kubernetes node
+- if expose the service via `LoadBalancer`, set the `domain` as your own domain name and add a CNAME record to map the domain name to the one you got from the cloud provider
+
+If Harbor is deployed behind the proxy, set it as the URL of proxy.
+
+#### Configure the way how to persistent data:
+
+- **Disable**: The data does not survive the termination of a pod.
+- **Persistent Volume Claim(default)**: A default `StorageClass` is needed in the Kubernetes cluster to dynamic provision the volumes. Specify another StorageClass in the `storageClass` or set `existingClaim` if you have already existing persistent volumes to use.
+- **External Storage(only for images and charts)**: For images and charts, the external storages are supported: `azure`, `gcs`, `s3` `swift` and `oss`.
+
+#### Configure the secrets
+
+- **Secret keys**: Secret keys are used for secure communication between components. Fill `core.secret`, `jobservice.secret` and `registry.secret` to configure.
+- **Certificates**:
```

Comment:
If there's only one certificate, then it may not make sense to have a bullet list